proto/frostdb/schema/v1alpha1/schema.proto

```diff
@@ -20,6 +20,8 @@ message Column {
     StorageLayout storage_layout = 2;
     // Whether the column can dynamically expand.
     bool dynamic = 3;
+    // Prehash the column before storing it. This is an optimization to speed up aggregation queries when this column is often aggregated. 
```

Comment:
nit: specify where the prehash is stored (i.e. a separate non-dynamic column)

Reply:
done.